src/sentry/demo/data_population.py

```diff
@@ -466,20 +508,23 @@ def fix_measurements(event_json):
         measurements.update(measurement_markers)
 
 
-def update_context(event, trace=None):
+def update_context(event, trace=None, mobile=False, platform=None):
     context = event["contexts"]
     # delete device since we aren't mocking it (yet)
-    if "device" in context:
+    if "device" in context and not mobile:
         del context["device"]
     # generate random browser and os
-    context.update(**gen_base_context())
+    if mobile:
+        device, os = gen_mobile_context(platform)
```

Comment:
@roggenkemper would be better if we could keep the api the same as `gen_base_context` so this becomes:
```
base_context = gen_mobile_context(platform) if mobile else gen_base_context()
context.update(**base_context)
```

Reply:
makes sense!